code_samples/api/product_catalog/src/Command/CatalogCommand.php

```diff
@@ -0,0 +1,90 @@
+<?php declare(strict_types=1);
+
+namespace App\Command;
+
+use Ibexa\Contracts\Core\Repository\PermissionResolver;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Ibexa\Contracts\ProductCatalog\CatalogServiceInterface;
+use Ibexa\Contracts\ProductCatalog\ProductServiceInterface;
+use Ibexa\Contracts\ProductCatalog\Values\Catalog\CatalogCreateStruct;
+use Ibexa\Contracts\ProductCatalog\Values\Catalog\CatalogUpdateStruct;
+use Ibexa\Contracts\ProductCatalog\Values\Product\ProductQuery;
+use Ibexa\Contracts\ProductCatalog\Values\Product\Query\Criterion;
+use Ibexa\ProductCatalog\Local\Repository\Values\Catalog\Status;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class CatalogCommand extends Command
+{
+    private UserService $userService;
+
+    private PermissionResolver $permissionResolver;
+
+    private ProductServiceInterface $productService;
+
+    private CatalogServiceInterface $catalogService;
+
+    public function __construct(UserService $userService, PermissionResolver $permissionResolver, ProductServiceInterface $productService, CatalogServiceInterface $catalogService)
```

Comment:
You could improve readability by putting each parameter in separate line

Please apply this suggestion to all commands.